.github/workflows/release.yml

```diff
@@ -148,6 +148,38 @@ jobs:
           name: azureauth-${{ github.event.inputs.version }}-${{ matrix.runtime }}
           path: azureauth-${{ github.event.inputs.version }}-${{ matrix.runtime }}
 
+  # Build and sign Linux binaries on Azure DevOps and publish them to GitHub and packages.microsoft.com.

```

Comment:
@goagain on line 42, we build azureauth for linux which currently doesn't build or create any debian package. We can skip doing that because we are building the package on ADO pipeline. I made that change in the PR #211 (https://github.com/AzureAD/microsoft-authentication-cli/blob/ed3cd91207f9b4180d0cc995a54b32492eb9c291/.github/workflows/release.yml#L42C1-L42C1)

Reply:
Removed linux build